src/lib.rs

```diff
@@ -565,4 +637,70 @@ mod tests {
         ],
         eval: { 3.0 => 155.324961718789 },
     }
+
+    test_beat! {
+        name: less_than,
```

Comment:
All the other tests have the same name as the operator, but in lower case, it might be better to name test tests as `lt`, `gt`, etc.